web/themes/custom/usagov/templates/field--node--field-custom-twig-content-funcionarios-electos.html.twig

```diff
@@ -59,7 +59,6 @@
                     name="input-state"
                     id="input-state"
                     required
-                    autocomplete="off"
```

Comment:
Check whether `autocomplete="off"` really should be removed here as well.

Reply:
See: https://github.com/usagov/usagov-2021/pull/2257#discussion_r2033962220